bin/dotd

```diff
@@ -129,7 +130,9 @@ EOS
 user_name = ask_for_name
 
 should_i "morning DTT" do
-  open_pull_request 'test', 'staging', 'DTT (Staging > Test)'
+  GitHub.create_pull_request(
+    base: 'test', head: 'staging', title: 'DTT (Staging > Test)'
+  )
```

Comment:
Might be worth polling; personally I don't evaluate a DTT PR for correctness before I merge anymore.  But I do see the argument that if we're removing guardrails like this, we should put better ones up (checking dev room topic).

If you'd like to proceed conservatively let's at least open the browser though.